worker/src/env.ts

```diff
@@ -49,6 +49,11 @@ const EnvSchema = z.object({
     .number()
     .positive()
     .default(20),
+  LANGFUSE_INGESTION_SECONDARY_QUEUE_PROCESSING_CONCURRENCY: z.coerce
+    .number()
+    .positive()
+    .default(20),
```

Comment:
Maybe we should go with a number like 5 here. I believe we set the one above to 8 for cloud so going as high as 20 seems odd.